languages/bash.scm

```diff
@@ -22,23 +24,43 @@
   "fi"
   (string)
   (test_command)
+  ; TODO: etc.
 ] @prepend_space
 
 ;; Commands
 
-; One command per line, modulo many exceptions:
-; * Strings of && and ||
-; * Pipelines
-; * Subshells
-; * As arguments to if, while, for, etc.
-; * Others?...
+; NOTE "Command" is shorthand for a "unit of execution":
+; * Simple commands (e.g., binaries, builtins, functions, etc.)
+; * Command lists
+; * Command pipelines
 ;
-; FIXME Can this be generalised, or does *every* context need to be
-; individually enumerated?...
-(program (command) @prepend_spaced_softline @append_hardline)
-(if_statement _ "then" (command) @prepend_spaced_softline)
-(elif_clause _ "then" (command) @prepend_spaced_softline)
-(else_clause (command) @prepend_spaced_softline)
+; That is: [(command) (list) (pipeline)], per the grammar
+
+; FIXME I don't think it's possible to insert the necessary line
+; continuations; or, at least, it's not possible to insert them only in
+; a multi-line context. As such, all multi-line commands are forcibly
+; collapsed on to a single line for now... See Issue #172
+
+; One command per line in the following contexts:
+; * Top-level
+; * In any branch of a conditional
+; * <TODO: etc.>
+;
+; NOTE Because "command" is such a pervasive and general concept, each
+; context needs to be individually enumerated to account for exceptions.
+(program [(command) (list) (pipeline)] @prepend_hardline)
```

Comment:
Even if it's more space-consuming, I think I prefer the heavily-indented way to write queries, even simple ones.
At least for consistency within the query files, e.g.
```
(program
  [
    (command)
    (list)
    (pipeline)
  ] @prepend_hardline
)
```
Or at least
```
(program
  [(command) (list) (pipeline)] @prepend_hardline
)
```
if the `[(command) (list) (pipeline)]` syntagm is expected to be everywhere

Reply:
Good idea :+1: I'll fix this in #174 to avoid making merge conflict problems for myself